akka-stream/src/main/scala/akka/stream/OverflowStrategy.scala

```diff
@@ -88,7 +88,7 @@ private[akka] object OverflowStrategies {
    * INTERNAL API
    */
   private[akka] case object EmitEarly extends DelayOverflowStrategy {
-    private[akka] override def isBackpressure: Boolean = false
+    private[akka] override def isBackpressure: Boolean = true
```

Comment:
It's not really no-backpressure, it can't pull and emit early unless downstream is ready

Reply:
Hmm, while I get it is 'not no-backpressure' this still seems a bit counter-intuitive... otoh without it the condition below would be something like `!isBackpressure && (s != EmitEarly || isAvailable(out))` which is not too readable either. OK since it's private :+1: